server/test/check.go

```diff
@@ -145,18 +145,8 @@ var checkQueryTests = []checkQueryTest{
 		typeDefinitions: []*openfgapb.TypeDefinition{{
 			Type: "repo",
 			Relations: map[string]*openfgapb.Userset{
-				"reader": {
-					Userset: &openfgapb.Userset_ComputedUserset{
-						ComputedUserset: &openfgapb.ObjectRelation{
-							Relation: "writer",
-						},
-					}},
-				"writer": {
-					Userset: &openfgapb.Userset_ComputedUserset{
-						ComputedUserset: &openfgapb.ObjectRelation{
-							Relation: "reader",
-						},
-					}},
+				"reader": ComputedUserset("", "writer"),
```

Comment:
Refactored these so they are easier to read and take up way less space. The helpers are in `server/test/helpers.go`

Reply:
i'm not against these changes but they belong to a different PR :P

Reply:
There are a lot more of these changes to make, and I plan on saving those for a future PR. This just started that work and made it a little easier for me to write and read the new tests I was writing, so I went ahead and did it just for `check.go`.